apps/foundation-app/src/app/cart/components/cart/cart.component.html

```diff
@@ -1,3 +1,7 @@
-<div class="cart-container">
-  {{cart.id}}
+<div class="cart">
```

Comment:
- [ ] What should this look like when the cart is empty? @xelaint let's see if we can't do a simple animation here.

- [ ] What about when we're loading cart items? @xelaint @lderrickable I'd determine whether or not there's a foundation loader, and use that. Otherwise, if we have to roll our own, let's pick one early, because we're definitely going to need it

Reply:
@lderrickable added design to zeplin for `empty cart view`.

Reply:
I think we're best off with font-awesome for the loader: https://fontawesome.com/icons/spinner?style=solid, but @xelaint can make the final call on that. Foundation doesn't have one

Reply:
Yes, the font-awesome loader works.

Reply:
added loading spinner for cart